### PR TITLE
Add docs preset/layout

### PR DESCRIPTION
**Pull request by oysteinsigholt**

### CODE REVIEW

File: src/docs-preset/components/base.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    errSelectors: PropTypes.object.isRequired,
+    errActions: PropTypes.object.isRequired,
+    specActions: PropTypes.object.isRequired,
+    specSelectors: PropTypes.object.isRequired,
+    layoutSelectors: PropTypes.object.isRequired,
+    layoutActions: PropTypes.object.isRequired,
+    getComponent: PropTypes.func.isRequired
+  }
+
+  onFilterChange =(e) => {
```
Comment (jorunfa) [inline src/docs-preset/components/base.jsx]: `=(e)` ->
`= (e)`

File: src/docs-preset/components/base.jsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+    return (
+
+      <div className='swagger-ui'>
+          <div>
+            <Errors/>
+            <Row className="information-container">
+              <Col mobile={12}>
+                { info.count() ? (
+                  <DocsInfo info={ info } url={ url } host={ host } basePath={ basePath } externalDocs={externalDocs} getComponent={getComponent}/>
```
Comment (jorunfa) [inline src/docs-preset/components/base.jsx]: There is a difference in the styling in `externalDocs` and `getComponent` from  `info`, `url`, and `basePath`. Why?

File: src/docs-preset/components/info.jsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  }
+}
+
+
+class Contact extends React.Component {
+  static propTypes = {
+    data: PropTypes.object
+  }
+
+  render(){
```
Comment (jorunfa) [inline src/docs-preset/components/info.jsx]: `render(){` ->
`render() {`

File: src/docs-preset/components/info.jsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    )
+  }
+}
+
+class License extends React.Component {
+  static propTypes = {
+    license: PropTypes.object
+  }
+
+  render(){
```
Comment (jorunfa) [inline src/docs-preset/components/info.jsx]: `render(){` ->
`render() {`

File: src/docs-preset/components/base.jsx (hunk 2 above)
Comment (oysteinsigholt) [inline src/docs-preset/components/base.jsx]: Copy paste from swagger ui, so just an inconsistency I guess. Should be fixed now

**Pull request merged**